src/main/views/sign-in.njk

```diff
@@ -54,12 +54,14 @@
         ]
       }) }}
         {{ submitButton(continueButton) }}
-      <div class="govuk-grid-row govuk-!-margin-bottom-2">
-        <div class="no_padding govuk-grid-column-full">
-          <p class="govuk-body-s govuk-!-font-weight-bold govuk-!-margin-bottom-1">{{ accountInfo }}</p>
-          <a class="govuk-link" href="{{ accountInfo_link }}">{{ accountInfo_link_text }}</a>
-        </div>
-      </div>
+
```

Comment:
Can I double check we're OK leaving the route accessible too? Or should the route also be disabled

Reply:
Checked with NC and leaving route accessible is fine, only want it temp removed off page

Reply:
Ok great :)